part-2-server/server.js

```diff
@@ -0,0 +1,31 @@
+var http = require('http')
+var fs = require('fs')
+
+var server = http.createServer(function(request, response){
+	var url = request.url
+	request.on("data", function(data1){
+		console.log(data1.toString());
+	})
+	if (url === "/") {
+		fs.readFile("index.html", function(err, data){
+			response.write(data.toString());
+			response.end();
+		})
+	
+	}else if (url === "/page2") {
+		fs.readFile("page-2.html", function(err,data){
+			response.write(data.toString());
+			response.end();
+		})
+	}else if (url ==="/poem-"){
+		fs.readFile("createPoemFiles.js", function(err,data){
```

Comment:
What is `i` referring to? The `i` variable is never declared anywhere. We can't use `i` outside of a for loop unless we first declare it somewhere.